compressor/tests/test_offline.py

```diff
@@ -72,10 +72,8 @@ def setUp(self):
         # TEMPLATES[1] to be Jinja2 templates backend in test_settings.
         TEMPLATES = copy.deepcopy(settings.TEMPLATES)
 
-        django_template_dir = os.path.join(
-            TEMPLATES[0]['DIRS'][0], self.templates_dir)
-        jinja2_template_dir = os.path.join(
-            TEMPLATES[1]['DIRS'][0], self.templates_dir)
+        django_template_dir = os.path.join(TEMPLATES[0]['DIRS'][0], self.templates_dir)
+        jinja2_template_dir = os.path.join(TEMPLATES[1]['DIRS'][0], self.templates_dir)
```

Comment:
Why change this ? It's now breaking 79 chars limit, which was the reason for the current wrapping (I know all compressor code doesn't follow pep8, but let's not regress on the files that do)